stages.md

```diff
@@ -0,0 +1,70 @@
+# Stages for changes to GraphQL specification
+
+This document describes set of stages required for merging changes to the
+GraphQL specification. These stages are intended to:
++ Be as flexible as possible.
++ Define a clear process for getting changes into the specification.
++ Provide transparency for participants in the GraphQL ecosystem of the
+status/progress of proposals.
++ Encourage community discussion around the proposed changes.
+
+Any changes to a specification that affect behavior of **GraphQL server or
+client implementation** should go through bellow stages no matter how small
+it is. All other changes should be labeled as “editorial” and could be merged
+right away.
+
+## Stage -2: proposed change(optional)
```

Comment:
I think we want whitespace before "(optional)" here, right? Same for the next stage heading as well.